src/components/Card/Card.scss

```diff
@@ -7,11 +7,19 @@
 
   + .Card {
     margin-top: spacing();
+
+    @media print {
+      margin-top: -1 * spacing(tight);
+    }
```

Comment:
Do you have an example of why the negative margin is needed for print?

Reply:
What about other areas of the admin where there is a list of cards, like the home page. Would that be negatively affected?

Reply:
This is a good point. In that case it is not affected because, cards are wrapped with `CardList_Item` thus `Card + Card` won't apply. I checked in the code and here are not many places where that can happen and in those case it will look like the screenshot attached only when printing, otherwise it won't apply either.